inc/category.class.php

```diff
@@ -471,9 +478,10 @@ public static function canApplyFilter($itilcategories_id) {
 
 
    static function getOthersGroupsID($level = 0) {
+      /** @var \DBmysql $DB */
       global $DB;
 
-      $res = $DB->query("SELECT gr.id
+      $res = $DB->doQuery("SELECT gr.id
```

Comment:
This should have been replaced with a properly built query, no a `doQuery` call.